### PR TITLE
[bn] Updated files with  `/bn`

### PR DESCRIPTION
**Pull request by nazma1234**
Added `/bn` with the links for Bengali Localization
Tracking Issue: #410

### CODE REVIEW

**Comment by mitul3737** [comment]
@jihoon-seo Can you have a look into these files?

File: content/bn/application_programming_interface.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ category: প্রযুক্তি
...
 
 ## এটি যেই সমস্যাটি দৃষ্টিপাত করে
-অ্যাপ্লিকেশনগুলি আরও জটিল হয়ে উঠলে, ছোট কোড পরিবর্তনগুলি অন্যান্য কার্যকারিতার উপর কঠোর প্রভাব ফেলতে পারে। অ্যাপ্লিকেশনগুলিকে তাদের কার্যকারিতার জন্য একটি মডুলার পদ্ধতি অবলম্বন করতে হবে যদি তারা একই সাথে বৃদ্ধি এবং স্থিতিশীলতা বজায় রাখতে পারে। API ছাড়া, অ্যাপ্লিকেশনগুলির মধ্যে মিথস্ক্রিয়া করার জন্য একটি কাঠামোর অভাব রয়েছে। একটি শেয়ার্ড ফ্রেমওয়ার্ক ছাড়া, অ্যাপ্লিকেশনগুলির জন্য [স্কেল(scale)](/scalability/) এবং একীভূত করা চ্যালেঞ্জিং।
+অ্যাপ্লিকেশনগুলি আরও জটিল হয়ে উঠলে, ছোট কোড পরিবর্তনগুলি অন্যান্য কার্যকারিতার উপর কঠোর প্রভাব ফেলতে পারে। অ্যাপ্লিকেশনগুলিকে তাদের কার্যকারিতার জন্য একটি মডুলার পদ্ধতি অবলম্বন করতে হবে যদি তারা একই সাথে বৃদ্ধি এবং স্থিতিশীলতা বজায় রাখতে পারে। API ছাড়া, অ্যাপ্লিকেশনগুলির মধ্যে মিথস্ক্রিয়া করার জন্য একটি কাঠামোর অভাব রয়েছে। একটি শেয়ার্ড ফ্রেমওয়ার্ক ছাড়া, অ্যাপ্লিকেশনগুলির জন্য [স্কেল(scale)](/bn/scalability/) এবং একীভূত করা চ্যালেঞ্জিং।
```
Comment (jihoon-seo) [inline content/bn/application_programming_interface.md:11]: In the `content/bn/` directory of `dev-bn` branch,
`scalability.md` file does not exist yet,
so this link should go to the English version of `Scalability` page.

```suggestion
অ্যাপ্লিকেশনগুলি আরও জটিল হয়ে উঠলে, ছোট কোড পরিবর্তনগুলি অন্যান্য কার্যকারিতার উপর কঠোর প্রভাব ফেলতে পারে। অ্যাপ্লিকেশনগুলিকে তাদের কার্যকারিতার জন্য একটি মডুলার পদ্ধতি অবলম্বন করতে হবে যদি তারা একই সাথে বৃদ্ধি এবং স্থিতিশীলতা বজায় রাখতে পারে। API ছাড়া, অ্যাপ্লিকেশনগুলির মধ্যে মিথস্ক্রিয়া করার জন্য একটি কাঠামোর অভাব রয়েছে। একটি শেয়ার্ড ফ্রেমওয়ার্ক ছাড়া, অ্যাপ্লিকেশনগুলির জন্য [স্কেল(scale)](/scalability/) এবং একীভূত করা চ্যালেঞ্জিং।
```

File: content/bn/application_programming_interface.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ category: প্রযুক্তি
...
 
 ## এটা কিভাবে সাহায্য করে
-APIগুলি কম্পিউটার প্রোগ্রাম বা অ্যাপ্লিকেশনগুলিকে একটি সংজ্ঞায়িত এবং বোধগম্য পদ্ধতিতে তথ্য আদান-প্রদান এবং আদান-প্রদান করার অনুমতি দেয়। তারা আধুনিক অ্যাপ্লিকেশনের জন্য বিল্ডিং ব্লক এবং তারা ডেভেলপারদের অ্যাপ্লিকেশন একত্রিত করার একটি উপায় প্রদান করে থাকে। যখনই আপনি [মাইক্রসার্ভিস(microservices)](/microservices/) একসাথে কাজ করার কথা শুনেন, আপনি অনুমান করতে পারেন যে তারা একটি API এর মাধ্যমে ইন্টারঅ্যাক্ট করে।
+APIগুলি কম্পিউটার প্রোগ্রাম বা অ্যাপ্লিকেশনগুলিকে একটি সংজ্ঞায়িত এবং বোধগম্য পদ্ধতিতে তথ্য আদান-প্রদান এবং আদান-প্রদান করার অনুমতি দেয়। তারা আধুনিক অ্যাপ্লিকেশনের জন্য বিল্ডিং ব্লক এবং তারা ডেভেলপারদের অ্যাপ্লিকেশন একত্রিত করার একটি উপায় প্রদান করে থাকে। যখনই আপনি [মাইক্রসার্ভিস(microservices)](/bn/microservices/) একসাথে কাজ করার কথা শুনেন, আপনি অনুমান করতে পারেন যে তারা একটি API এর মাধ্যমে ইন্টারঅ্যাক্ট করে।
```
Comment (jihoon-seo) [inline content/bn/application_programming_interface.md:14]:
```suggestion
APIগুলি কম্পিউটার প্রোগ্রাম বা অ্যাপ্লিকেশনগুলিকে একটি সংজ্ঞায়িত এবং বোধগম্য পদ্ধতিতে তথ্য আদান-প্রদান এবং আদান-প্রদান করার অনুমতি দেয়। তারা আধুনিক অ্যাপ্লিকেশনের জন্য বিল্ডিং ব্লক এবং তারা ডেভেলপারদের অ্যাপ্লিকেশন একত্রিত করার একটি উপায় প্রদান করে থাকে। যখনই আপনি [মাইক্রসার্ভিস(microservices)](/microservices/) একসাথে কাজ করার কথা শুনেন, আপনি অনুমান করতে পারেন যে তারা একটি API এর মাধ্যমে ইন্টারঅ্যাক্ট করে।
```

File: content/bn/cloud_computing.md [hunk 3]
```diff
@@ -11,4 +11,4 @@ category: ধারণা
...
 
 ## এটা কিভাবে সাহায্য করে
-ক্লাউড সুবিধা প্রদানকারী সংস্থাসমূহ তাদের ব্যবহারকারী সংস্থাগুলিকে অর্থের বিনিময়ে চাহিদা অনুযায়ী কম্পিউট রিসোর্স ভাড়া করার এবং ব্যবহার করার ক্ষমতা প্রদান করে। এটি দুটি প্রধান উদ্ভাবনের অনুমতি দেয়: সংস্থাগুলি ভৌত অবকাঠামোতে অর্থ বা সংস্থান ব্যয় না করে এবং সময় অপচয় না করে নতুন কিছু চেষ্টা করতে পারে এবং তারা প্রয়োজন এবং চাহিদা অনুযায়ী [স্কেল(scale)](/scalability/) করতে পারে। ক্লাউড সুবিধা প্রদানকারী সংস্থাসমূহ তাদের ব্যবহারকারী সংস্থাগুলিকে প্রয়োজন অনুযায়ী বা সর্বনিম্ন প্রয়োজন মোতাবেক পরিকাঠামো ব্যবহার করতে দেয়।
+ক্লাউড সুবিধা প্রদানকারী সংস্থাসমূহ তাদের ব্যবহারকারী সংস্থাগুলিকে অর্থের বিনিময়ে চাহিদা অনুযায়ী কম্পিউট রিসোর্স ভাড়া করার এবং ব্যবহার করার ক্ষমতা প্রদান করে। এটি দুটি প্রধান উদ্ভাবনের অনুমতি দেয়: সংস্থাগুলি ভৌত অবকাঠামোতে অর্থ বা সংস্থান ব্যয় না করে এবং সময় অপচয় না করে নতুন কিছু চেষ্টা করতে পারে এবং তারা প্রয়োজন এবং চাহিদা অনুযায়ী [স্কেল(scale)](/bn/scalability/) করতে পারে। ক্লাউড সুবিধা প্রদানকারী সংস্থাসমূহ তাদের ব্যবহারকারী সংস্থাগুলিকে প্রয়োজন অনুযায়ী বা সর্বনিম্ন প্রয়োজন মোতাবেক পরিকাঠামো ব্যবহার করতে দেয়।
```
Comment (jihoon-seo) [inline content/bn/cloud_computing.md:14]:
```suggestion
ক্লাউড সুবিধা প্রদানকারী সংস্থাসমূহ তাদের ব্যবহারকারী সংস্থাগুলিকে অর্থের বিনিময়ে চাহিদা অনুযায়ী কম্পিউট রিসোর্স ভাড়া করার এবং ব্যবহার করার ক্ষমতা প্রদান করে। এটি দুটি প্রধান উদ্ভাবনের অনুমতি দেয়: সংস্থাগুলি ভৌত অবকাঠামোতে অর্থ বা সংস্থান ব্যয় না করে এবং সময় অপচয় না করে নতুন কিছু চেষ্টা করতে পারে এবং তারা প্রয়োজন এবং চাহিদা অনুযায়ী [স্কেল(scale)](/scalability/) করতে পারে। ক্লাউড সুবিধা প্রদানকারী সংস্থাসমূহ তাদের ব্যবহারকারী সংস্থাগুলিকে প্রয়োজন অনুযায়ী বা সর্বনিম্ন প্রয়োজন মোতাবেক পরিকাঠামো ব্যবহার করতে দেয়।
```

File: content/bn/cloud_native_security.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ category: ধারণা
 
 ## এটা কি
 
-ক্লাউড নেটিভ সিকিউরিটি এমন একটি পদ্ধতি যা [ক্লাউড নেটিভ অ্যাপ্লিকেশন](/cloud_native_apps/) এ নিরাপত্তা তৈরি করে। এটি নিশ্চিত করে যে নিরাপত্তা উন্নয়ন থেকে উৎপাদন পর্যন্ত সমগ্র অ্যাপ্লিকেশন জীবনচক্রের অংশ। ক্লাউড নেটিভ সিকিউরিটি ক্লাউড নেটিভ এনভায়রনমেন্টের বিবরণ, যথা দ্রুত কোড পরিবর্তন এবং অত্যন্ত ক্ষণস্থায়ী অবকাঠামোর সাথে খাপ খাওয়ানোর সময় প্রথাগত নিরাপত্তা মডেলের মতো একই মান নিশ্চিত করতে চায়। ক্লাউড নেটিভ নিরাপত্তা [DevSecOps](/devsecops/) নামক অনুশীলনের সাথে অত্যন্ত সম্পর্কিত।
+ক্লাউড নেটিভ সিকিউরিটি এমন একটি পদ্ধতি যা [ক্লাউড নেটিভ অ্যাপ্লিকেশন](/bn/cloud_native_apps/) এ নিরাপত্তা তৈরি করে। এটি নিশ্চিত করে যে নিরাপত্তা উন্নয়ন থেকে উৎপাদন পর্যন্ত সমগ্র অ্যাপ্লিকেশন জীবনচক্রের অংশ। ক্লাউড নেটিভ সিকিউরিটি ক্লাউড নেটিভ এনভায়রনমেন্টের বিবরণ, যথা দ্রুত কোড পরিবর্তন এবং অত্যন্ত ক্ষণস্থায়ী অবকাঠামোর সাথে খাপ খাওয়ানোর সময় প্রথাগত নিরাপত্তা মডেলের মতো একই মান নিশ্চিত করতে চায়। ক্লাউড নেটিভ নিরাপত্তা [DevSecOps](/bn/devsecops/) নামক অনুশীলনের সাথে অত্যন্ত সম্পর্কিত।
```
Comment (jihoon-seo) [inline content/bn/cloud_native_security.md:9]:
```suggestion
ক্লাউড নেটিভ সিকিউরিটি এমন একটি পদ্ধতি যা [ক্লাউড নেটিভ অ্যাপ্লিকেশন](/cloud_native_apps/) এ নিরাপত্তা তৈরি করে। এটি নিশ্চিত করে যে নিরাপত্তা উন্নয়ন থেকে উৎপাদন পর্যন্ত সমগ্র অ্যাপ্লিকেশন জীবনচক্রের অংশ। ক্লাউড নেটিভ সিকিউরিটি ক্লাউড নেটিভ এনভায়রনমেন্টের বিবরণ, যথা দ্রুত কোড পরিবর্তন এবং অত্যন্ত ক্ষণস্থায়ী অবকাঠামোর সাথে খাপ খাওয়ানোর সময় প্রথাগত নিরাপত্তা মডেলের মতো একই মান নিশ্চিত করতে চায়। ক্লাউড নেটিভ নিরাপত্তা [DevSecOps](/devsecops/) নামক অনুশীলনের সাথে অত্যন্ত সম্পর্কিত।
```

File: content/bn/cloud_native_tech.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ category: ধারণা
 
 ## এটা কি
 
-ক্লাউড নেটিভ টেকনোলজি, ক্লাউড নেটিভ স্ট্যাক হিসেবেও উল্লেখ করা হয়, [ক্লাউড নেটিভ অ্যাপ্লিকেশন](/cloud_native_apps/) তৈরি করতে ব্যবহৃত প্রযুক্তি। সরকারী, প্রাইভেট এবং হাইব্রিড ক্লাউডের মতো আধুনিক, গতিশীল পরিবেশে মাপযোগ্য অ্যাপ্লিকেশনগুলি তৈরি এবং চালানোর জন্য সংস্থাগুলিকে সক্ষম করে, তারা 'ক্লাউডের প্রতিশ্রুতি' বজায় রাখে এবং ক্লাউড কম্পিউটিং সুবিধাগুলি তাদের সম্পূর্ণরূপে লাভ করে। ক্লাউড কম্পিউটিং এবং কন্টেইনার, সার্ভিস মেশ, মাইক্রোসার্ভিসেস এবং অপরিবর্তনীয় অবকাঠামোর ক্ষমতাকে কাজে লাগানোর জন্য গ্রাউন্ড আপ থেকে ডিজাইন করা হয়েছে এই পদ্ধতির উদাহরণ।
+ক্লাউড নেটিভ টেকনোলজি, ক্লাউড নেটিভ স্ট্যাক হিসেবেও উল্লেখ করা হয়, [ক্লাউড নেটিভ অ্যাপ্লিকেশন](/bn/cloud_native_apps/) তৈরি করতে ব্যবহৃত প্রযুক্তি। সরকারী, প্রাইভেট এবং হাইব্রিড ক্লাউডের মতো আধুনিক, গতিশীল পরিবেশে মাপযোগ্য অ্যাপ্লিকেশনগুলি তৈরি এবং চালানোর জন্য সংস্থাগুলিকে সক্ষম করে, তারা 'ক্লাউডের প্রতিশ্রুতি' বজায় রাখে এবং ক্লাউড কম্পিউটিং সুবিধাগুলি তাদের সম্পূর্ণরূপে লাভ করে। ক্লাউড কম্পিউটিং এবং কন্টেইনার, সার্ভিস মেশ, মাইক্রোসার্ভিসেস এবং অপরিবর্তনীয় অবকাঠামোর ক্ষমতাকে কাজে লাগানোর জন্য গ্রাউন্ড আপ থেকে ডিজাইন করা হয়েছে এই পদ্ধতির উদাহরণ।
```
Comment (jihoon-seo) [inline content/bn/cloud_native_tech.md:9]:
```suggestion
ক্লাউড নেটিভ টেকনোলজি, ক্লাউড নেটিভ স্ট্যাক হিসেবেও উল্লেখ করা হয়, [ক্লাউড নেটিভ অ্যাপ্লিকেশন](/cloud_native_apps/) তৈরি করতে ব্যবহৃত প্রযুক্তি। সরকারী, প্রাইভেট এবং হাইব্রিড ক্লাউডের মতো আধুনিক, গতিশীল পরিবেশে মাপযোগ্য অ্যাপ্লিকেশনগুলি তৈরি এবং চালানোর জন্য সংস্থাগুলিকে সক্ষম করে, তারা 'ক্লাউডের প্রতিশ্রুতি' বজায় রাখে এবং ক্লাউড কম্পিউটিং সুবিধাগুলি তাদের সম্পূর্ণরূপে লাভ করে। ক্লাউড কম্পিউটিং এবং কন্টেইনার, সার্ভিস মেশ, মাইক্রোসার্ভিসেস এবং অপরিবর্তনীয় অবকাঠামোর ক্ষমতাকে কাজে লাগানোর জন্য গ্রাউন্ড আপ থেকে ডিজাইন করা হয়েছে এই পদ্ধতির উদাহরণ।
```

File: content/bn/cluster.md [hunk 6]
```diff
@@ -6,18 +6,18 @@ category: ধারণা
 
 ## এটা কি
 
-একটি ক্লাস্টার হল কম্পিউটার বা অ্যাপ্লিকেশনগুলির একটি গ্রুপ যা একটি সাধারণ লক্ষ্যে একসাথে কাজ করে। ক্লাউড নেটিভ কম্পিউটিং প্রসঙ্গে, শব্দটি প্রায়শই কুবারনেটে প্রয়োগ করা হয়। একটি Kubernetes ক্লাস্টার হল পরিষেবাগুলির একটি সেট (বা কাজের চাপ) যা তাদের নিজস্ব পাত্রে চলে, সাধারণত বিভিন্ন মেশিনে। এই সমস্ত [কন্টেইনারাইজড(Contanerrized)](/containerization/) পরিষেবাগুলির সংগ্রহ, একটি নেটওয়ার্কের মাধ্যমে সংযুক্ত, একটি ক্লাস্টার প্রতিনিধিত্ব করে।
+একটি ক্লাস্টার হল কম্পিউটার বা অ্যাপ্লিকেশনগুলির একটি গ্রুপ যা একটি সাধারণ লক্ষ্যে একসাথে কাজ করে। ক্লাউড নেটিভ কম্পিউটিং প্রসঙ্গে, শব্দটি প্রায়শই কুবারনেটে প্রয়োগ করা হয়। একটি Kubernetes ক্লাস্টার হল পরিষেবাগুলির একটি সেট (বা কাজের চাপ) যা তাদের নিজস্ব পাত্রে চলে, সাধারণত বিভিন্ন মেশিনে। এই সমস্ত [কন্টেইনারাইজড(Contanerrized)](/bn/containerization/) পরিষেবাগুলির সংগ্রহ, একটি নেটওয়ার্কের মাধ্যমে সংযুক্ত, একটি ক্লাস্টার প্রতিনিধিত্ব করে।
```
Comment (jihoon-seo) [inline content/bn/cluster.md:9]:
```suggestion
একটি ক্লাস্টার হল কম্পিউটার বা অ্যাপ্লিকেশনগুলির একটি গ্রুপ যা একটি সাধারণ লক্ষ্যে একসাথে কাজ করে। ক্লাউড নেটিভ কম্পিউটিং প্রসঙ্গে, শব্দটি প্রায়শই কুবারনেটে প্রয়োগ করা হয়। একটি Kubernetes ক্লাস্টার হল পরিষেবাগুলির একটি সেট (বা কাজের চাপ) যা তাদের নিজস্ব পাত্রে চলে, সাধারণত বিভিন্ন মেশিনে। এই সমস্ত [কন্টেইনারাইজড(Containerized)](/containerization/) পরিষেবাগুলির সংগ্রহ, একটি নেটওয়ার্কের মাধ্যমে সংযুক্ত, একটি ক্লাস্টার প্রতিনিধিত্ব করে।
```

File: content/bn/cluster.md [hunk 7]
```diff
@@ -6,18 +6,18 @@ category: ধারণা
...
 
 
 
 ## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
 
-একটি একক কম্পিউটারে চলা সফ্টওয়্যার ব্যর্থতার একটি একক পয়েন্ট উপস্থাপন করে — যদি সেই কম্পিউটারটি ক্র্যাশ হয়ে যায়, বা কেউ দুর্ঘটনাক্রমে পাওয়ার কেবলটি আনপ্লাগ করে, তবে কিছু ব্যবসা-সংক্রান্ত সমস্যা  সিস্টেম অফলাইনে নেওয়া হতে পারে। এই কারণেই আধুনিক সফ্টওয়্যারগুলি সাধারণত [ডিস্ট্রিবিউটেড অ্যাপ্লিকেশন(Distributed application)](/distributed_apps/) হিসাবে তৈরি করা হয়, ক্লাস্টার হিসাবে একসাথে গ্রুপ করা হয়।
+একটি একক কম্পিউটারে চলা সফ্টওয়্যার ব্যর্থতার একটি একক পয়েন্ট উপস্থাপন করে — যদি সেই কম্পিউটারটি ক্র্যাশ হয়ে যায়, বা কেউ দুর্ঘটনাক্রমে পাওয়ার কেবলটি আনপ্লাগ করে, তবে কিছু ব্যবসা-সংক্রান্ত সমস্যা  সিস্টেম অফলাইনে নেওয়া হতে পারে। এই কারণেই আধুনিক সফ্টওয়্যারগুলি সাধারণত [ডিস্ট্রিবিউটেড অ্যাপ্লিকেশন(Distributed application)](/bn/distributed_apps/) হিসাবে তৈরি করা হয়, ক্লাস্টার হিসাবে একসাথে গ্রুপ করা হয়।
```
Comment (jihoon-seo) [inline content/bn/cluster.md:15]:
```suggestion
একটি একক কম্পিউটারে চলা সফ্টওয়্যার ব্যর্থতার একটি একক পয়েন্ট উপস্থাপন করে — যদি সেই কম্পিউটারটি ক্র্যাশ হয়ে যায়, বা কেউ দুর্ঘটনাক্রমে পাওয়ার কেবলটি আনপ্লাগ করে, তবে কিছু ব্যবসা-সংক্রান্ত সমস্যা  সিস্টেম অফলাইনে নেওয়া হতে পারে। এই কারণেই আধুনিক সফ্টওয়্যারগুলি সাধারণত [ডিস্ট্রিবিউটেড অ্যাপ্লিকেশন(Distributed application)](/distributed_apps/) হিসাবে তৈরি করা হয়, ক্লাস্টার হিসাবে একসাথে গ্রুপ করা হয়।
```

File: content/bn/software_as_a_service.md [hunk 8]
```diff
@@ -10,7 +10,7 @@ Category: প্রযুক্তি
...
+প্রথাগতভাবে, ব্যবসায়িক সফ্টওয়্যারগুলো পৃথক কম্পিউটারে ইনস্টল করা হয়, যার রক্ষণাবেক্ষণ এবং আপডেট করার জন্য একজন প্রশাসকের প্রয়োজন হয়। উদাহরণ স্বরূপ: একটি প্রতিষ্ঠান গ্রাহক চাহিদা ব্যবস্থাপনা (CRM) এর জন্য স্ব-শরীর(on-premise) সফ্টওয়্যার ব্যবহার করতে পারে। এই সফ্টওয়্যারটি অভ্যন্তরীণ আইটি বিভাগ নিয়োগ করে ক্রয়, ইনস্টল, সুরক্ষা, রক্ষণাবেক্ষণ এবং নিয়মিত আপগ্রেড করা প্রয়োজন, যা আইটি টিমের উপর একটি বোঝাস্বরূপ ৷ লাইসেন্স, ইন্সটলেশন এবং সম্ভাব্য অতিরিক্ত হার্ডওয়্যারের সাথে যুক্ত আপ ফ্রন্ট খরচ নিষিদ্ধ হতে পারে। চাহিদার প্রতি সাড়া দেওয়াও কঠিন হতে পারে এবং [scale](/bn/scalability/) বৃদ্ধি বা পরিবর্তনের প্রতিক্রিয়ায় দ্রুত প্রয়োজন অনুযায়ী উপরে ও নিচে যাতায়াত সম্ভব না হতে পারে।
```
Comment (jihoon-seo) [inline content/bn/software_as_a_service.md:13]:
```suggestion
প্রথাগতভাবে, ব্যবসায়িক সফ্টওয়্যারগুলো পৃথক কম্পিউটারে ইনস্টল করা হয়, যার রক্ষণাবেক্ষণ এবং আপডেট করার জন্য একজন প্রশাসকের প্রয়োজন হয়। উদাহরণ স্বরূপ: একটি প্রতিষ্ঠান গ্রাহক চাহিদা ব্যবস্থাপনা (CRM) এর জন্য স্ব-শরীর(on-premise) সফ্টওয়্যার ব্যবহার করতে পারে। এই সফ্টওয়্যারটি অভ্যন্তরীণ আইটি বিভাগ নিয়োগ করে ক্রয়, ইনস্টল, সুরক্ষা, রক্ষণাবেক্ষণ এবং নিয়মিত আপগ্রেড করা প্রয়োজন, যা আইটি টিমের উপর একটি বোঝাস্বরূপ ৷ লাইসেন্স, ইন্সটলেশন এবং সম্ভাব্য অতিরিক্ত হার্ডওয়্যারের সাথে যুক্ত আপ ফ্রন্ট খরচ নিষিদ্ধ হতে পারে। চাহিদার প্রতি সাড়া দেওয়াও কঠিন হতে পারে এবং [scale](/scalability/) বৃদ্ধি বা পরিবর্তনের প্রতিক্রিয়ায় দ্রুত প্রয়োজন অনুযায়ী উপরে ও নিচে যাতায়াত সম্ভব না হতে পারে।
```

**Review by jihoon-seo — COMMENTED** [review]
I left some comments.

**Comment by jihoon-seo** [comment]
Dear Bengali approvers (@mitul3737 @Mouly22 @ikramulkayes @Imtiaz1234):
please use "Squash and merge" when merging PRs!

**Pull request closed without merging**